<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-46E6TOF" failures="1" tests="3" name="Test" time="49.146" errors="0" timestamp="2022-07-08T15:03:01 IST">
  <testcase classname="Sample.campaignTest" name="CreateCampaignTest" time="8.118"/>
  <testcase classname="Sample.campaignTest" name="createCampaignWithproductTest" time="19.913"/>
  <testcase classname="Sample.OrganizationTest" name="CreateNewOrganizationTest" time="3.477">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Sample.OrganizationTest.CreateNewOrganizationTest(OrganizationTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- CreateNewOrganizationTest -->
  <testcase classname="com.crm.GHYT15.GenericUtility.BaseClass" name="@AfterSuite confiCloseDB" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite confiCloseDB -->
  <testcase classname="com.crm.GHYT15.GenericUtility.BaseClass" name="@BeforeSuite confiConnectDB" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite confiConnectDB -->
</testsuite> <!-- Test -->
